packages/e2e-test-utils-playwright/src/test.ts

```diff
@@ -140,7 +140,7 @@ const test = base.extend<
 			} );
 
 			await Promise.all( [
-				requestUtils.activateTheme( 'twentytwentyone' ),
+				// requestUtils.activateTheme( 'twentytwentyone' ),
```

Comment:
This is probably an oversight?

Reply:
Oops, yes, was debugging why this util didn't work for me.